bin/scripts/export-data/getAllEventsSegmentsForApp.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Output list of app names, event keys, and segments.
+ * Server: countly
+ * Path: bin/scripts/export-data
+ * Command: node getallEventsSegmentsForApp.js
+ */
+
+var plugins = require("../../../plugins/pluginManager.js");
+
+plugins.dbConnection("countly").then(function(db) {
+    db.collection("events").find().toArray(function(err, events) {
+        db.collection("apps").find().toArray(function(err, names) {
```

Comment:
Here from app documents we only use name property, so let's provide it in projection, so we read from disk only that property and not whole document and not consume too much RAM to store whole app object there

Reply:
I've modified the code to specify only the name property! :)